doc/modules/mod_http_upload.md

```diff
@@ -14,17 +14,18 @@ Currently, the module supports only the S3 backend using [AWS Signature Version
 * **token_bytes** (integer, default: `32`) - Number of random bytes of a token that will be used in a generated URL. 
  The text representation of the token will be twice as long as the number of bytes, e.g. for the default value the token in URL will be 64 characters long.
 * **max_file_size** (integer, default: 10485760 (10 MB)) - Maximum file size (in bytes) accepted by the module. Disabled if set to `undefined`.
-* **s3** (list, default: unset) - Options specific to S3 backend.
+* **s3** (list, default: unset) - Options specific to [S3][s3] backend.
 
-#### S3 backend options
+#### [S3][s3] backend options
 
 * **bucket_url** (string, default: unset) - A complete URL pointing at the used bucket. The URL may be in [virtual host form][aws-virtual-host], and for AWS needs to point at a specific regional endpoint for the bucket. The scheme, port and path specified in the URL will be used to create `PUT` URLs for slots, e.g. specifying a value of `"https://s3-eu-west-1.amazonaws.com/mybucket/custom/prefix"` will result in `PUT` URLs of form `"https://s3-eu-west-1.amazonaws.com/mybucket/custom/prefix/<RANDOM_TOKEN>/<FILENAME>?<AUTHENTICATION_PARAMETERS>"`.
```

Comment:
```suggestion
* **bucket_url** (string, default: unset) - A complete URL pointing at the used bucket. The URL may be in [virtual host form][aws-virtual-host], and for AWS it needs to point to a specific regional endpoint for the bucket. The scheme, port and path specified in the URL will be used to create `PUT` URLs for slots, e.g. specifying a value of `"https://s3-eu-west-1.amazonaws.com/mybucket/custom/prefix"` will result in `PUT` URLs of form `"https://s3-eu-west-1.amazonaws.com/mybucket/custom/prefix/<RANDOM_TOKEN>/<FILENAME>?<AUTHENTICATION_PARAMETERS>"`.
```
Yeah, I know this wasn't touched by the PR, but now that I see it anyway 🤷 
_Point at_ is more of a physical gesture, while _point to_ is more direction-wise.

Reply:
well, this PR is about changing this file, so fill free to recheck all the rest parts